src/components/Shared/CrossingMapPage/CrossingMapSearchBar.scss

```diff
@@ -94,8 +94,7 @@
 .react-autosuggest__suggestions-container {
   color: black;
   background: #f8f8f9;
-  margin-top: 15px;
-  margin-right: -31px;
+  border-top: 10px solid #2c579b;
```

Comment:
lots of random hexes floating around in this code... would be great to consolidate these in a variables file.